nile-backend/src/main/java/com/tp/Nile/repositories/ProductRepository.java

```diff
@@ -13,8 +13,7 @@
 
 @Repository
 public interface ProductRepository extends JpaRepository<Product,Integer> {
-
-    @Query("select p from Product p where p.category_id=:category")
```

Comment:
The issue with the query is that Product is referring to the class of Product not the the table in the database. Instead of p.category_id, it should be p.category